### DISCUSSION TITLE
Documentation Navbar not showing

### DISCUSSION THREAD

**Discussion by Surfix** [giswqs/geodemo#3] [Q&A]
Hi

The navbar of my package documentation is not showing. The gh-page appear to be ok and it displays the index.md perfectly but no navbar. I appreciate any suggestion in fixing this.

**Comment by giswqs** [comment]
Please share links to your GitHub repo and docs website

**Comment by Surfix** [comment]
Check the repo [here](https://github.com/Surfix/Icechart) and the documentation website is available [here](https://surfix.github.io/Icechart/)

**Reply by giswqs** [reply]
Change the branch in docs.yml from master to main

https://github.com/Surfix/Icechart/blob/main/.github/workflows/docs.yml

**Reply by Surfix** [reply]
Thank you for the suggestion. I have tried this but the nav bar is still not showing. I have confirmed that the documentation rendered the README.md from the docs folder but the nav is not showing. I even used Jekyll theme but still the same.  I must have missed something.

**Reply by giswqs** [reply]
You can take a look at the error: https://github.com/Surfix/Icechart/runs/2061665251?check_suite_focus=true
Your notebook appears to be causing the problem. Try deleting the notebook.

**Reply by Surfix** [reply]
Hi Prof, I have diagnosed the package using your suggestion but the navbar is still not showing. I noticed that the document page is rendered properly if served locally using "mkdocs serve" command (see first image) but the online GitHub page would not show well  (second image).  Thanks for your selfless service.

![image](https://user-images.githubusercontent.com/51313944/113229757-2c58d380-92ca-11eb-97c9-31b4697e96b3.png)

![image](https://user-images.githubusercontent.com/51313944/113229956-95404b80-92ca-11eb-9856-9e0cef3674a4.png)

**Reply by giswqs** [reply]
Did you change your repo GitHub Pages to use the branch `gh-pages`?

![image](https://user-images.githubusercontent.com/5016453/113230431-f9312880-9266-11eb-9e0b-81e76cb12904.png)

**Reply by Surfix** [reply]
I did just that and it rendered properly.  Thanks for the guide.
